server/svix-server/src/queue/redis.rs

```diff
@@ -588,6 +594,31 @@ impl TaskQueueReceive for RedisQueueConsumer {
     }
 }
 
+async fn clear_acked(pool: RedisPool, queue: &str) -> Result<()> {
+    let mut conn = ctx!(pool.get().await)?;
+
+    let xpending_resp: StreamPendingReply =
+        ctx!(conn.query_async(Cmd::xpending(queue, WORKERS_GROUP)).await)?;
```

Comment:
You want to add a count (limit) to the number your return, you only want 1 anyway...